pkg/api/v1/schema.go

```diff
@@ -217,6 +217,9 @@ type Devices struct {
 	// Whether to have random number generator from host
 	// +optional
 	Rng *Rng `json:"rng,omitempty"`
+	// Whether or not to enable virtio multi-queue
+	// +optional
+	MultiQueue *bool `json:"multiQueue,omitempty"`
```

Comment:
Are we planning to use this parameter for enabling __all__ virtio queues? (virtio-scsi, virtio-net?)
So if a VM has block and scsi disks and virtio nics will all enable MQ?

Reply:
Oh you know what, you mentioned that before and I forgot. I'll change the name, as this applies specifically to disks.

thanks @vladikr!